src/main/java/com/training/assignments/Operations.java

```diff
@@ -0,0 +1,98 @@
+package com.training.assignments;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import java.util.HashMap;
+
+public class Operations  {
+    private Logger logger = LoggerFactory.getLogger(this.getClass());
+
+    public String printCompanyReviews(String companyName){
+        HashMap<String,String> company=new HashMap<>();
+        company.put("Endee engineers","good company for production of portable and online instrument of gas analyzer");
+        company.put("Aaion automation","installation of GPS tracker");
+        company.put("Thermolab","good company for production of stabilizers and incubator");
+        company.put("Shreeji solution and services","provide solutions of fire alarm system");
+        company.put("Google","Best platform where u will get solution for everything");
+        logger.info("You have entered: "+companyName+":");
+        switch(companyName)
+        {
+            case "Endee engineers":
+                logger.info("good company for production of portable and online instrument of gas analyzer");
+                break;
+            case "Aaion automation":
+                logger.info("installation of GPS tracker");
+                break;
+            case "Thermolab":
+                logger.info("good company for production of stabilizers and incubator");
+                break;
+            case "Shreeji solution and services":
+                logger.info("provide solutions of fire alarm system");
+                break;
+            case "Google":
+                logger.info("Best platform where u will get solution for everything");
```

Comment:
What is the use of printing hardcoded values in logger? 
Instead print logger.info(company.get(companyName)) after the switch construct.